src/scripts/app.js

```diff
@@ -1 +1,29 @@
-console.log('hello');
+/* eslint-disable no-unused-vars */
+// eslint-disable-next-line no-undef
+const slider = tns({
+  container: '.my-slider',
```

Comment:
możemy stworzyć jakiś globalny selektor, po dodaniu którego zawszę będzie slider? np `tsn-slider`. W tedy możemy bazować na atrybutach `data-tsn-*` do konfiguracji naszego slidera? Np mamy element, który ma być sliderem i mieć `4` itemy na start to dodajemy atrybut `data-tsn-item="4"` a w JS:
```
items: el.dataset('tsn-items') || 4
```

Reply:
Ok, muszę dodać gdzieś te atrybuty? Chyba nie w html? Nie rozumiem i nie wiem jak mam to poprawić :(

Reply:
ogólnie tak - powinnaś zmienić to w HTML również tzn mamy np nasz slider:
```
<div class="my-slider" data-tsn-items="4" data-inna-opcja="inna-wartosc">
  <div></div>
  <div></div>
  <div></div>
</div>
```

i teraz w JS - łapiemy wszystkie elementy `my-slider` i robimy coś takiego:
```
const sliders = document.querySelectorAll('.my-slider');

for (let slider of sliders) {
    tsn({
        container: slider,
        items: slider.dataset('tsn-items') || 4, // tutaj jest informacja o tym, że jeśli nie będzie w HTML tej wartości to weźmie 4
        // ... inne opcje
    })
}
```

Reply:
to pozwoli nam na generowanie sliderów bez konieczności kolejnego kombinowania w JS